### PR TITLE
Cherry picking from gm/conv_patch_tracking

### PR DESCRIPTION
**Pull request by eightysteele**
This PR is a candidate to replace #109. I copied in the files on top of the latest `main` commit. All tests pass.

### CODE REVIEW

**Comment by georgematheos** [comment]
I'll give Mirko a chance to glance at the FeatureTrackData changes, and I'll merge this tomorrow.  I will fix the paths loaded in the notebooks in a future PR (since they are just loaded in the notebooks right now I won't worry about passing through main to do this).

**Review by georgematheos — APPROVED** [review]

**Comment by georgematheos** [comment]
This looks good to me!  @eightysteele I don't actually know how to add a commit onto this type of PR made from a cherry-pick, but I can just fix the asset path issue you caught in the notebooks after we have merged this into main, in another PR.  I will give @mirkoklukas a chance to glance at the changes in FeatureTrackData if he wants to, and will merge this tomorrow.

**Comment by eightysteele** [comment]
Sounds good! I _think_ you can just checkout this branch like normal and commit changes directly to it? Nothing special required since we're both collaborators on the repo with write access. Otherwise a follow up PR sgtm.

File: src/b3d/io/feature_track_data.py
Comment (mirkoklukas) [inline src/b3d/io/feature_track_data.py:1]: think in 466 the division by 255 should go as well.

**Comment by georgematheos** [comment]
I fixed the error @mirkoklukas pointed out and the paths @eightysteele, and have merged.  Thanks for the help with this!

**Pull request merged**
